core/src/main/java/org/apache/iceberg/MergingSnapshotProducer.java

```diff
@@ -125,6 +127,16 @@ public ThisT set(String property, String value) {
     return self();
   }
 
+  protected void caseSensitiveBinding(boolean newCaseSensitive) {
+    this.caseSensitive = newCaseSensitive;
+    filterManager.caseSensitive(newCaseSensitive);
+    deleteFilterManager.caseSensitive(newCaseSensitive);
+  }
+
+  protected boolean caseSensitive() {
```

Comment:
I think this should be `isCaseSensitive`. I think we mostly use the `is` prefix for returning booleans like this.

Reply:
Fixed.